src/test/kotlin/no/nav/syfo/testutil/mock/IsproxyMock.kt

```diff
@@ -14,16 +14,18 @@ import no.nav.syfo.testutil.UserConstants.VIRKSOMHETSNUMMER_DEFAULT
 import no.nav.syfo.testutil.UserConstants.VIRKSOMHETSNUMMER_NO_VIRKSOMHETSNAVN
 import no.nav.syfo.testutil.getRandomPort
 
+val eregOrganisasjonResponse = EregOrganisasjonResponse(
+    navn = EregOrganisasjonNavn(
+        navnelinje1 = "Virksom Bedrift AS",
+        redigertnavn = "Virksom Bedrift AS, Norge",
+    )
+)
+
 class IsproxyMock {
     private val port = getRandomPort()
     val url = "http://localhost:$port"
 
-    val eregOrganisasjonResponse = EregOrganisasjonResponse(
-        navn = EregOrganisasjonNavn(
-            navnelinje1 = "Virksom Bedrift AS",
-            redigertnavn = "Virksom Bedrift AS, Norge",
-        )
-    )
+    val response = eregOrganisasjonResponse
```

Comment:
Er det noen grunn til å lage en `respons` her, i stedet for å bruke `eregOrganisasjonResponse` direkte på linje 38?

Reply:
Nei, jeg skal endre det 👍